public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -112,6 +112,7 @@ function getDataSourceModalStyles(theme: GrafanaTheme2) {
     modal: css`
       width: 80%;
```

Comment:
Think we need a max-width, it is huuuuge on large screens:

<img width="2998" alt="image" src="https://user-images.githubusercontent.com/2376619/233292981-9b3c3132-39a9-4f3d-8217-2b15f12ce03d.png">

Reply:
Fixed